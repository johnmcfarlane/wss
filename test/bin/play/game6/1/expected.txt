 60  9,1  | MOHELS
 57  9,1  | OLEUMS
 54  9,1  | HOLMES
 54  9,1  | MOUSLE
 30  9,3  | MOHELS
 30  6,4  | MUDHOLE
 29  9,5  | HEMOL
 29  9,3  | MOHEL
 28  9,3  | MOUSLE
 27  9,2  | HOLMES
 27  9,3  | HOMES
 27  9,4  | SHEMU
 27  9,6  | SHEMU
 26  9,5  | HELMS
 26  9,5  | HEML
 26  9,3  | HUME
 26  9,3  | HUMS
 26  6,4  | MODULES
 26  9,3  | MULES
 26  9,3  | MULSE
 26  6,1  | MUSHED
 25  9,2  | EMULS
 25  9,6  | EMULS
 25  9,2  | HELMS
 25  9,3  | HEMS
 25  9,5  | HEMS
 25  9,2  | HOLMS
 25  9,3  | HOME
 25  9,3  | HOUSEL
 25  9,5  | MELOS
 25  9,3  | MOLES
 25  9,3  | MOSEL
 25  9,3  | MOUES
 25  9,3  | MOUSE
 25  9,4  | OLEUMS
 25  9,4  | SHEM
 25  9,6  | SHEM
 24  6,6  - DEVELS
 24  9,5  | HELM
 24  9,5  | HEM
 24  9,3  | LUMS
 24  9,5  | MEH
 24  6,2  | MOULDS
 24  9,3  | MULE
 24  9,4  | MUSH
 24  9,6  | SLUM
 23  9,3  | HOLES
 23  9,3  | HOSEL
 23  9,3  | HOUSE
 23  9,2  | HOUSEL
 23  8,4  | HOVELS
 23  9,3  | MELS
 23  9,5  | MELS
 23  9,5  | MESH
 23  9,3  | MHOS
 23  9,3  | MOLE
 23  9,3  | MOLS
 23  9,3  | MOUE
 23  9,3  | OHMS
 23  9,4  | OLEUM
 23  8,4  | SEVUM
 23  8,3  | SHOVEL
 22  6,6  - DEVEL
 22  6,6  | DUMOSE
 22  9,3  | ELMS
 22  9,6  | EMUS
 22  6,2  | HOMED
 22  8,4  | HOVEL
 22  9,4  | LUSH
 22  6,1  | LUSHED
 22  9,5  | MEL
 22  9,4  | MESH
 22  9,5  | MEU
 22  9,4  | MOSUL
 22  6,2  | MOULD
 22  9,2  | MOULE
 22  9,2  | MOULS
 22  6,1  | MOUSED
 22  8,4  | MOVES
 22  9,5  | MSL
 22  6,2  | MULED
 22  9,2  | MULES
 22  9,2  | MULSE
 22  9,3  | MUSE
 22  9,6  | SHLU
 22  9,6  | SHMO
 22  9,2  | SHOLE
 22  9,6  | SHOLE
 22  6,1  | SHOULD
 22  9,6  | SHUL
 22  9,6  | SOLUM
 22  9,6  | SOUM
 21  9,3  | HOLE
 21  9,3  | HOLS
 21  9,2  | HOMES
 21  9,3  | LOUSE
 21  9,5  | MESO
 21  3,7  - MOHEL
 21  9,3  | MOSE
 21  3,7  - MOSUL
 21  9,4  | SHEL
 21  9,6  | SHEL
 21  9,4  | SHEOL
 21  9,6  | SHEOL
 21  9,6  | SHOU
 21  9,3  | SOME
 20  6,6  | DHOLES
 20  6,6  | DHOUL
 20  6,6  | DOMUS
 20  9,6  | ELMS
 20  9,6  | EMU
 20  9,3  | EMUS
 20  9,5  | HEL
 20  3,7  - HEMOL
 20  6,1  | HOUSED
 20  9,3  | HUES
 20  9,3  | HUSE
 20  9,4  | LOSH
 20  9,2  | MELOS
 20  9,5  | MEO
 20  9,5  | MES
 20  9,2  | MOLES
 20  9,4  | MOSEL
 20  9,2  | MOUES
 20  9,2  | MOUSE
 20  9,4  | MUSE
 20  6,2  | MUSED
 20  9,4  | MUSO
 20  9,3  | OUSEL
 20  8,5  | OVUM
 20  9,5  | SEM
 20  9,5  | SEOUL
 20  9,6  | SML
 20  9,6  | SUMO
 20  9,5  | USH
 19  9,6  | EMO
 19  9,4  | EMS
 19  9,6  | EMS
 19  9,4  | EUSOL
 19  9,6  | EUSOL
 19  9,3  | HOES
 19  9,2  | HOLES
 19  9,3  | HOSE
 19  9,4  | HOSEL
 19  9,2  | HOUSE
 19  9,4  | HUES
 19  9,4  | HUSE
 19  9,4  | HUSO
 19  3,7  - MOSEL
 19  8,4  | MOVE
 19  3,7  - MULES
 19  9,4  | OMS
 19  9,4  | OUSEL
 19  9,6  | SEOUL
 19  9,6  | SHOE
 19  8,6  | VOLUME
 19  8,6  | VUM
 18  6,6  | DEMOS
 18  6,6  | DHOLE
 18  6,6  | DHOLS
 18  6,6  | DOMES
 18  9,6  | ELM
 18  9,5  | HEO
 18  9,5  | HES
 18  6,2  | HOLED
 18  2,7  - HOLMES
 18  3,7  - HOMES
 18  9,5  | LEU
 18  9,2  | LOUSE
 18  6,1  | LOUSED
 18  9,3  | LUES
 18  9,4  | LUES
 18  9,5  | ME
 18  3,7  - MELOS
 18  9,4  | MESO
 18  8,4  | MEV
 18  3,7  - MOLES
 18  9,4  | MOSE
 18  3,7  - MOUES
 18  9,4  | MUS
 18  6,2  | SHELD
 18  2,7  - SHEMU
 18  9,6  | SHU
 18  9,3  | SLUE
 18  9,6  | SLUE
 18  9,6  | SOME
 18  9,6  | SOUL
 18  6,1  | SOULED
 18  9,6  | SUM
 18  9,4  | UHS
 18  8,6  | VELUM
 17  7,1  | HEMULE
 17  9,4  | HOES
 17  9,4  | HOSE
 17  3,7  - HOSEL
 17  2,7  - HOUSEL
 17  9,4  | HUE
 17  9,3  | LEOS
 17  9,5  | LEOS
 17  9,3  | LOSE
 17  9,4  | OHS
 17  9,4  | OLES
 17  9,4  | SHE
 17  9,6  | SHE
 17  4,5  - SHEM
 17  9,6  | SHO
 17  9,3  | SHOE
 17  8,3  | SHOVE
 17  9,6  | SLOE
 17  9,3  | SOLE
 16  6,6  | DOLUS
 16  6,6  | DUELS
 16  6,6  | DULSE
 16  6,6  | DUOLE
 16  9,6  | EH
 16  9,5  | ESU
 16  9,5  | HE
 16  5,5  - HEM
 16  3,7  - HOLES
 16  5,5  - HOM
 16  6,2  | HOSED
 16  8,4  | HOVE
 16  3,5  - HUME
 16  3,7  - HUME
 16  9,5  | LEO
 16  9,4  | LOSE
 16  8,4  | LOVES
 16  9,4  | LUE
 16  9,4  | MES
 16  7,3  | MOHELS
 16  9,4  | MOS
 16  4,7  - MOUL
 16  1,5  - MOUSLE
 16  1,7  - MOUSLE
 16  3,5  - MULE
 16  3,7  - MULE
 16  9,3  | OLES
 16  8,5  | OVULES
 16  9,5  | SEL
 16  9,6  | SEM
 16  9,6  | SH
 16  6,2  | SHOED
 16  9,3  | SLOE
 16  6,2  | SLUED
 16  9,6  | SOLE
 16  9,4  | ULE
 16  9,5  | USE
 15  9,4  | ELS
 15  9,6  | ELS
 15  7,5  | HEMULE
 15  9,4  | HES
 15  9,4  | HOE
 15  3,5  - HOME
 15  3,7  - HOME
 15  5,5  - MEH
 15  5,7  - MEH
 15  7,3  | MOHEL
 15  3,5  - MOLE
 15  3,7  - MOLE
 15  3,5  - MOUE
 15  3,7  - MOUE
 15  2,5  - MULSE
 15  2,7  - MULSE
 15  3,5  - MUSE
 15  3,7  - MUSE
 15  3,5  - MUSO
 15  3,7  - MUSO
 15  9,4  | OLE
 15  8,5  | OVULE
 15  3,7  - SEOUL
 15  3,7  - SHEOL
 15  4,7  - SHUL
 15  9,6  | SOH
 15  9,4  | SUE
 15  9,6  | SUE
 14  6,6  | DOLES
 14  6,6  | DOUSE
 14  9,6  | EL
 14  7,2  | EMULE
 14  4,7  - EMUS
 14  9,6  | ESU
 14  3,7  - EUSOL
 14  8,3  | HOUVE
 14  9,4  | LOE
 14  3,5  - MESO
 14  3,7  - MESO
 14  4,7  - MHOS
 14  3,5  - MOSE
 14  3,7  - MOSE
 14  4,7  - MOUE
 14  2,5  - MOULE
 14  2,7  - MOULE
 14  7,3  | MULES
 14  9,5  | OES
 14  9,5  | OSE
 14  3,7  - OUSEL
 14  9,6  | SEL
 14  5,5  - SEM
 14  7,5  | SEMOULE
 14  2,5  - SHOLE
 14  2,7  - SHOLE
 14  9,6  | SOL
 14  6,2  | SOLED
 14  9,6  | SOU
 14  3,5  - ULMO
 14  3,7  - ULMO
 14  9,5  | US
 14  9,4  | USE
 14  8,6  | VOLES
 13  7,3  | HEMES
 13  7,5  | HEMOL
 13  3,5  - HOLE
 13  3,7  - HOLE
 13  7,2  | HOLMES
 13  7,3  | HOMES
 13  7,3  | HUME
 13  3,5  - HUSE
 13  3,7  - HUSE
 13  3,5  - HUSO
 13  3,7  - HUSO
 13  8,4  | LEVO
 13  8,4  | LOVE
 13  5,7  - MEL
 13  7,3  | MELES
 13  7,3  | MESEL
 13  5,7  - MOL
 13  7,3  | MOLES
 13  7,3  | MOSEL
 13  7,3  | MOUES
 13  2,5  - MOUSE
 13  2,7  - MOUSE
 13  7,1  | MOUSLE
 13  7,3  | MULE
 13  5,7  - MUS
 13  9,4  | OES
 13  6,5  - OM
 13  9,4  | OSE
 13  4,7  - SHEL
 13  3,7  - SHLU
 13  3,5  - SHMO
 13  3,7  - SHMO
 13  9,4  | SOE
 13  9,6  | SOE
 13  4,7  - SOUL
 13  6,7  - UH
 13  8,6  | VELO
 13  8,6  | VOLE
 13  8,6  | VOLS
 12  7,6  | ELEM
 12  7,6  | EMULE
 12  7,6  | EMULS
 12  9,6  | EO
 12  9,5  | ES
 12  9,6  | ES
 12  5,7  - HEL
 12  7,5  | HELMS
 12  7,3  | HEME
 12  5,7  - HOL
 12  7,3  | HOME
 12  3,5  - HOSE
 12  3,7  - HOSE
 12  2,5  - HOUSE
 12  2,7  - HOUSE
 12  5,7  - HUE
 12  4,7  - HUES
 12  8,4  | LEV
 12  7,3  | MELE
 12  7,5  | MESOLE
 12  6,4  | MODULE
 12  7,3  | MOLE
 12  7,3  | MOUE
 12  7,1  | MOUSEE
 12  7,2  | MULSE
 12  7,3  | MUSE
 12  9,5  | OE
 12  7,4  | OLEUMS
 12  9,5  | OS
 12  3,7  - SHOU
 12  4,7  - SLUE
 12  9,6  | SO
 12  5,5  - SOH
 12  5,7  - SOH
 12  8,3  | SOLVE
 12  3,5  - SUMO
 12  3,7  - SUMO
 12  8,6  | VEL
 12  8,6  | VOL
 11  6,5  - EH
 11  6,7  - EH
 11  7,6  | EMEUS
 11  4,7  - EMU
 11  7,5  | HEMES
 11  4,7  - HOES
 11  7,3  | HOLES
 11  7,3  | HOSEL
 11  7,2  | HOUSEL
 11  6,4  | LEDUM
 11  2,5  - LOUSE
 11  2,7  - LOUSE
 11  5,7  - LUE
 11  4,7  - LUES
 11  5,7  - LUO
 11  5,5  - MEO
 11  5,7  - MEO
 11  5,7  - MES
 11  7,3  | MESE
 11  7,5  | MESEL
 11  7,1  | MESOLE
 11  4,7  - MEU
 11  4,5  - MHO
 11  4,7  - MHO
 11  6,4  | MODELS
 11  5,7  - MOS
 11  7,3  | MOSE
 11  4,7  - MOU
 11  7,2  | MOULE
 11  7,2  | MOUSEE
 11  6,5  - OH
 11  6,7  - OH
 11  6,3  | SELDOM
 11  7,5  | SEMEL
 11  7,4  | SHEMU
 11  4,5  - SHOE
 11  4,7  - SHOE
 11  7,2  | SHOLE
 11  6,3  | SOMDEL
 11  3,5  - SOME
 11  3,7  - SOME
 10  7,6  | EMEU
 10  7,3  | HELE
 10  7,5  | HELM
 10  7,5  | HEML
 10  5,5  - HEO
 10  5,7  - HEO
 10  5,7  - HES
 10  5,5  - HOE
 10  5,7  - HOE
 10  7,3  | HOLE
 10  7,3  | HUSE
 10  7,3  | LEME
 10  4,7  - LEOS
 10  3,5  - LOSE
 10  3,7  - LOSE
 10  6,3  | MELDS
 10  7,5  | MELES
 10  7,2  | MELOE
 10  7,5  | MELOE
 10  7,5  | MELOS
 10  7,2  | MEUSE
 10  7,5  | MEUSE
 10  6,4  | MODEL
 10  6,4  | MODUS
 10  6,3  | MOLDS
 10  7,2  | MOUSE
 10  5,7  - MU
 10  6,5  | ODEUM
 10  4,7  - OLES
 10  7,4  | OLEUM
 10  6,4  | SEDUM
 10  5,7  - SEL
 10  7,3  | SEMEL
 10  3,5  - SHOE
 10  3,7  - SHOE
 10  4,7  - SHU
 10  4,5  - SLOE
 10  4,7  - SLOE
 10  3,5  - SLUE
 10  3,7  - SLUE
 10  5,7  - SOL
 10  5,7  - SUE
 10  8,6  | VOES
  9  6,6  - DEVE
  9  6,6  - DEVS
  9  6,6  | DOUM
  9  7,6  | EHEU
  9  6,7  - EL
  9  7,6  | ELMS
  9  7,4  | EMEUS
  9  4,5  - EMO
  9  4,7  - EMO
  9  7,6  | EMUS
  9  7,5  | HEELS
  9  7,2  | HELOE
  9  7,5  | HELOE
  9  7,5  | HEME
  9  7,5  | HEMS
  9  6,3  | HOLDS
  9  7,3  | HOSE
  9  7,2  | HOUSE
  9  4,5  - HUE
  9  4,7  - HUE
  9  5,5  - LEO
  9  5,7  - LEO
  9  4,7  - LEU
  9  5,5  - LOE
  9  5,7  - LOE
  9  4,7  - LOU
  9  6,3  | MELD
  9  4,5  - MEO
  9  4,7  - MEO
  9  7,5  | MESH
  9  6,4  | MODES
  9  6,3  | MOLD
  9  6,3  | MOUD
  9  6,4  | MUDS
  9  6,5  | ODUM
  9  7,5  | SEOUL
  9  8,4  | SEV
  9  6,3  | SHEDU
  9  7,3  | SHEEL
  9  7,4  | SHEM
  9  3,5  - SLOE
  9  3,7  - SLOE
  9  3,5  - SOLE
  9  3,7  - SOLE
  9  8,4  | SOV
  9  6,5  | UDOM
  9  6,7  - US
  9  8,6  | VOE
  8  6,6  | DEMO
  8  6,6  | DOME
  8  6,6  | DOMS
  8  6,6  | DUM
  8  6,6  | DUSH
  8  7,4  | ELEM
  8  7,6  | EMES
  8  7,4  | EMEU
  8  4,7  - ESU
  8  7,6  | EUSOL
  8  7,4  | HEELS
  8  6,3  | HELD
  8  7,5  | HEM
  8  4,5  - HEO
  8  4,7  - HEO
  8  4,5  - HOE
  8  4,7  - HOE
  8  6,3  | HOLD
  8  6,3  | HUED
  8  7,5  | LEME
  8  6,3  | LEUDS
  8  7,2  | LOUSE
  8  5,7  - LU
  8  4,5  - LUE
  8  4,7  - LUE
  8  4,5  - LUO
  8  4,7  - LUO
  8  6,5  | MDSE
  8  5,5  - ME
  8  5,7  - ME
  8  7,5  | MEH
  8  7,5  | MELE
  8  7,5  | MELS
  8  5,5  - MO
  8  5,7  - MO
  8  6,4  | MODE
  8  6,4  | MODS
  8  6,3  | MOED
  8  6,4  | MUD
  8  5,7  - OES
  8  7,3  | OUSEL
  8  7,3  | SEME
  8  4,5  - SHE
  8  4,7  - SHE
  8  7,4  | SHEEL
  8  7,4  | SHEOL
  8  4,5  - SHO
  8  4,7  - SHO
  8  6,3  | SHODE
  8  7,3  | SMEE
  8  5,5  - SOE
  8  5,7  - SOE
  8  7,3  | SOME
  8  4,7  - SOU
  8  4,5  - ULE
  8  4,7  - ULE
  7  6,6  | DEM
  7  6,6  | DEUL
  7  6,6  | DHU
  7  6,6  | DOM
  7  6,6  | DUEL
  7  6,5  | EDHS
  7  7,4  | EHEU
  7  7,6  | ELM
  7  7,4  | EMES
  7  7,6  | EMU
  7  6,5  - EO
  7  6,7  - EO
  7  6,7  - ES
  7  5,5  - HE
  7  5,7  - HE
  7  7,4  | HEEL
  7  7,5  | HEEL
  7  7,5  | HELE
  7  6,4  | HLD
  7  5,5  - HO
  7  5,7  - HO
  7  6,4  | HODS
  7  6,3  | HOED
  7  6,4  | HUD
  7  7,4  | HUES
  7  4,5  - LEO
  7  4,7  - LEO
  7  7,3  | LESE
  7  6,3  | LEUD
  7  6,4  | LHD
  7  6,4  | LODES
  7  4,5  - LOE
  7  4,7  - LOE
  7  7,3  | LOSE
  7  6,3  | LOUD
  7  6,4  | LUDO
  7  6,4  | MED
  7  7,5  | MEL
  7  7,5  | MESE
  7  7,5  | MESO
  7  7,5  | MEU
  7  6,4  | MOD
  7  6,5  - OE
  7  6,7  - OE
  7  6,3  | OHED
  7  4,5  - OLE
  7  4,7  - OLE
  7  6,7  - OS
  7  7,4  | SEEM
  7  7,5  | SEEM
  7  7,5  | SEME
  7  6,3  | SHED
  7  7,3  | SHEE
  7  7,4  | SHEL
  7  6,3  | SHOD
  7  7,3  | SHOE
  7  6,3  | SLUD
  7  7,3  | SLUE
  7  7,4  | SMEE
  7  4,5  - SUE
  7  4,7  - SUE
  7  6,3  | SULD
  7  4,5  - USE
  7  4,7  - USE
  7  7,3  | USEE
  6  6,6  | DELS
  6  6,6  | DEUS
  6  6,6  | DOH
  6  6,6  | DOLE
  6  6,6  | DOLS
  6  6,6  | DUES
  6  6,6  | DUOS
  6  6,5  | EDH
  6  7,6  | EELS
  6  6,4  | ELDS
  6  6,3  | ELOD
  6  7,3  | ELSE
  6  7,6  | ELSE
  6  7,4  | EME
  6  7,6  | EME
  6  7,6  | EMO
  6  7,6  | EMS
  6  7,3  | EOLE
  6  7,6  | EOLE
  6  6,4  | HED
  6  7,5  | HEL
  6  6,4  | HOD
  6  7,4  | HOES
  6  7,4  | HUE
  6  6,4  | LEDS
  6  5,5  - LO
  6  5,7  - LO
  6  6,4  | LODE
  6  6,3  | LOED
  6  6,4  | LUD
  6  7,4  | LUES
  6  7,4  | MEE
  6  7,5  | MEE
  6  7,5  | MEO
  6  7,5  | MES
  6  6,5  | ODEL
  6  6,4  | OLDS
  6  4,5  - OSE
  6  4,7  - OSE
  6  6,4  | OUDS
  6  6,3  | SELD
  6  7,3  | SELE
  6  7,5  | SEM
  6  7,4  | SHEE
  6  6,3  | SLED
  6  7,3  | SLEE
  6  6,3  | SLOD
  6  7,3  | SLOE
  6  4,5  - SOE
  6  4,7  - SOE
  6  6,3  | SOLD
  6  7,3  | SOLE
  6  6,3  | SOUD
  6  6,4  | SUDE
  6  6,3  | SUED
  6  6,5  | UDOS
  6  6,3  | USED
  5  6,6  | DEL
  5  6,6  | DOES
  5  6,6  | DOL
  5  6,6  | DOSE
  5  6,6  | DUE
  5  6,6  | DUO
  5  7,5  | EELS
  5  6,4  | ELD
  5  5,5  - EO
  5  5,7  - EO
  5  7,4  | HEE
  5  7,5  | HEE
  5  7,5  | HEO
  5  7,5  | HES
  5  7,4  | HOE
  5  6,4  | LED
  5  7,4  | LEES
  5  7,5  | LEES
  5  7,5  | LEOS
  5  7,5  | LESE
  5  7,5  | LEU
  5  6,4  | LOD
  5  6,4  | LSD
  5  7,4  | LUE
  5  7,5  | ME
  5  6,5  | ODES
  5  5,5  - OE
  5  5,7  - OE
  5  6,4  | OLD
  5  7,4  | OLES
  5  6,4  | OUD
  5  7,4  | SEEL
  5  7,5  | SEEL
  5  7,5  | SELE
  5  7,4  | SHE
  5  6,4  | SLD
  5  7,4  | SLEE
  5  5,5  - SO
  5  5,7  - SO
  5  6,4  | SUD
  5  6,5  | UDO
  5  6,5  | UDS
  5  7,4  | ULE
  5  7,4  | USEE
  4  6,6  | DES
  4  6,6  | DOE
  4  6,6  | DOS
  4  6,6  | DU
  4  6,5  | EDO
  4  7,5  | EEL
  4  7,6  | EEL
  4  7,6  | EH
  4  7,6  | ELS
  4  6,4  | ESD
  4  7,6  | ESU
  4  7,5  | HE
  4  7,4  | LEE
  4  7,5  | LEE
  4  7,5  | LEO
  4  7,4  | LOE
  4  6,5  | ODE
  4  6,5  | ODS
  4  7,4  | OLE
  4  6,4  | SED
  4  7,5  | SEL
  4  6,4  | SOD
  4  7,4  | SUE
  4  7,4  | USE
  3  6,6  | DE
  3  6,6  | DO
  3  6,5  | ED
  3  7,6  | EL
  3  7,4  | ESE
  3  7,6  | ESE
  3  6,5  | OD
  3  7,5  | OES
  3  7,4  | OSE
  3  7,4  | SEE
  3  7,5  | SEE
  3  7,4  | SOE
  2  7,5  | EE
  2  7,6  | EE
  2  7,6  | EO
  2  7,6  | ES
  2  7,5  | OE
