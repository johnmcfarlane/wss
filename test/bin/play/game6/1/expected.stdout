8,0:mohels=60
8,0:oleums=57
5,3:mudholes=32
5,3:mudhole=30
8,2:hums=26
5,3:modules=26
5,0:mushed=26
8,1:helms=25
8,2:hems=25
8,1:holms=25
8,2:housel=25
8,2:mouse=25
5,5:devels=24
8,2:lums=24
5,0:moshed=24
5,1:moulds=24
8,3:mush=24
8,5:slum=24
8,2:house=23
7,3:hovels=23
8,2:mels=23
8,2:mhos=23
8,2:mols=23
8,2:ohms=23
7,2:shovel=23
5,5:devel=22
8,2:elms=22
5,1:homed=22
7,3:hovel=22
8,3:lush=22
5,0:lushed=22
8,3:mesh=22
8,3:mosh=22
5,1:mould=22
5,0:moused=22
7,3:moves=22
5,1:muled=22
8,1:mules=22
5,4:odeums=22
8,5:shmo=22
5,0:should=22
8,5:shul=22
8,5:solum=22
8,2:hols=21
8,1:homes=21
8,2:louse=21
8,1:meous=21
2,6:mohel=21
8,5:sheol=21
5,5:dholes=20
5,5:doums=20
8,2:emus=20
5,0:housed=20
8,2:hues=20
8,1:moles=20
8,1:moues=20
8,3:muse=20
5,1:mused=20
7,4:ovum=20
8,5:sumo=20
8,3:ems=19
8,1:helos=19
8,2:hoes=19
8,1:holes=19
8,3:hosel=19
7,3:move=19
2,6:mules=19
8,3:oms=19
8,3:ousel=19
8,5:shoe=19
7,5:volume=19
7,5:vum=19
5,5:demos=18
5,5:dhole=18
5,5:domes=18
5,1:holed=18
2,6:homes=18
5,0:loused=18
8,2:lues=18
2,6:moles=18
2,6:moues=18
8,3:mus=18
8,5:slue=18
8,5:some=18
8,5:soul=18
5,0:souled=18
8,5:sum=18
7,5:velum=18
8,3:hose=17
2,6:hosel=17
1,6:housel=17
8,3:ohs=17
8,5:she=17
7,2:shove=17
8,5:sloe=17
5,5:duels=16
5,5:dulse=16
2,6:helos=16
4,6:hem=16
4,4:hem=16
2,6:holes=16
5,1:hosed=16
7,3:hove=16
8,3:lose=16
7,3:loves=16
6,2:mohels=16
8,3:mos=16
2,6:mule=16
2,4:mule=16
8,2:oles=16
7,4:ovules=16
8,5:sh=16
5,1:shoed=16
5,1:slued=16
8,5:sole=16
8,5:som=16
8,4:use=16
8,3:els=15
8,3:hes=15
2,6:home=15
2,4:home=15
8,3:hos=15
6,2:mohel=15
2,6:mole=15
2,4:mole=15
2,4:moue=15
2,6:moue=15
2,4:muse=15
2,6:muse=15
7,4:ovule=15
2,6:sheol=15
8,5:sue=15
5,5:doles=14
5,5:douse=14
8,3:les=14
7,3:luvs=14
3,6:mhos=14
6,2:mules=14
8,4:ose=14
2,6:ousel=14
8,5:sel=14
8,5:sol=14
5,1:soled=14
4,4:som=14
4,6:som=14
8,5:sou=14
8,4:us=14
7,5:voles=14
5,4:em=13
5,6:em=13
2,4:helo=13
2,6:helo=13
6,2:hemes=13
2,6:hole=13
2,4:hole=13
6,2:homes=13
7,3:levo=13
7,3:love=13
7,3:luv=13
4,6:mel=13
4,6:mol=13
6,2:moles=13
6,2:moues=13
1,6:mouse=13
1,4:mouse=13
6,2:mule=13
8,3:oes=13
5,6:om=13
5,4:om=13
2,6:shmo=13
2,4:shmo=13
7,5:vole=13
8,4:es=12
6,4:helms=12
6,2:heme=12
6,2:home=12
2,4:hose=12
2,6:hose=12
1,6:house=12
1,4:house=12
3,6:hues=12
7,3:lev=12
5,3:module=12
6,2:mole=12
6,2:moue=12
6,2:muse=12
6,3:oleums=12
8,4:os=12
8,5:so=12
7,2:solve=12
2,4:sumo=12
2,6:sumo=12
5,6:eh=11
5,4:eh=11
6,5:emeus=11
6,4:hemes=11
3,6:hoes=11
6,2:holes=11
6,2:hosel=11
6,1:housel=11
1,6:louse=11
1,4:louse=11
3,6:lues=11
3,6:mho=11
3,4:mho=11
5,3:models=11
4,6:mos=11
5,6:oh=11
5,4:oh=11
5,2:seldom=11
2,4:some=11
2,6:some=11
6,5:emeu=10
6,4:helm=10
4,6:hes=10
6,2:hole=10
4,6:hos=10
2,6:lose=10
2,4:lose=10
5,2:melds=10
6,4:meous=10
5,3:model=10
5,3:modus=10
5,2:molds=10
6,1:mouse=10
5,4:odeum=10
3,6:oles=10
6,3:oleum=10
5,3:sedum=10
4,6:sel=10
2,4:shoe=10
2,6:shoe=10
2,4:slue=10
2,6:slue=10
4,6:sol=10
7,5:voes=10
5,5:devs=9
5,5:doum=9
5,6:el=9
6,5:elms=9
6,3:emeus=9
6,5:emus=9
6,4:heels=9
6,4:helos=9
6,4:heme=9
6,4:hems=9
5,2:holds=9
6,2:hose=9
6,1:house=9
3,6:hue=9
3,4:hue=9
4,6:les=9
5,2:meld=9
6,4:mesh=9
5,3:modes=9
5,2:mold=9
5,3:muds=9
2,6:sloe=9
2,4:sloe=9
2,6:sole=9
2,4:sole=9
7,5:voe=9
5,5:demo=8
5,5:dome=8
5,5:doms=8
6,5:emes=8
6,3:emeu=8
6,3:heels=8
5,2:held=8
6,4:hem=8
3,4:hoe=8
3,6:hoe=8
5,2:hold=8
5,2:hued=8
5,2:leuds=8
6,1:louse=8
5,3:ludes=8
4,6:me=8
4,4:me=8
5,3:meds=8
6,4:mels=8
6,4:meou=8
4,4:mo=8
4,6:mo=8
5,3:mode=8
5,3:mods=8
5,3:mud=8
4,6:oes=8
6,2:ousel=8
6,2:seme=8
3,4:she=8
3,6:she=8
6,3:sheol=8
6,2:some=8
5,5:dom=7
5,5:duel=7
5,5:duh=7
5,4:edhs=7
6,5:elm=7
6,3:emes=7
6,5:emu=7
5,6:es=7
4,6:he=7
4,4:he=7
6,3:heel=7
6,4:heel=7
6,4:helo=7
4,6:ho=7
4,4:ho=7
5,3:hods=7
5,2:hoed=7
6,3:hues=7
5,2:leud=7
5,3:lodes=7
6,2:lose=7
5,2:loud=7
5,3:lude=7
5,3:med=7
6,4:mel=7
5,3:mod=7
5,2:ohed=7
3,6:ole=7
3,4:ole=7
5,6:os=7
6,4:seem=7
6,3:seem=7
6,4:seme=7
5,2:shed=7
5,2:shod=7
6,2:shoe=7
6,2:slue=7
3,4:sue=7
3,6:sue=7
3,6:use=7
3,4:use=7
5,5:dels=6
5,5:dole=6
5,5:dols=6
5,5:dues=6
5,5:duos=6
5,4:edh=6
6,5:eels=6
5,3:elds=6
6,5:else=6
6,2:else=6
6,5:eme=6
6,3:eme=6
6,5:ems=6
5,3:hod=6
6,3:hoes=6
6,3:hue=6
4,4:lo=6
4,6:lo=6
5,3:lode=6
6,3:lues=6
5,3:olds=6
3,6:ose=6
3,4:ose=6
5,3:ouds=6
5,2:sled=6
6,2:sloe=6
5,2:sold=6
6,2:sole=6
5,2:sued=6
5,4:udos=6
5,2:used=6
5,5:del=5
5,5:does=5
5,5:dol=5
5,5:dose=5
5,5:due=5
5,5:duo=5
6,4:eels=5
5,3:eld=5
6,5:em=5
6,4:hes=5
6,3:hoe=5
5,3:led=5
6,3:lees=5
6,4:lees=5
6,4:leu=5
6,4:me=5
5,4:odes=5
4,4:oe=5
4,6:oe=5
5,3:old=5
6,3:oles=5
5,3:oud=5
6,4:seel=5
6,3:seel=5
6,3:she=5
4,6:so=5
4,4:so=5
5,4:udo=5
5,5:doe=4
5,5:dos=4
5,4:eds=4
6,5:eel=4
6,4:eel=4
6,5:eh=4
6,5:els=4
6,4:he=4
6,3:lee=4
6,4:lee=4
6,4:les=4
5,4:ode=4
5,4:ods=4
6,3:ole=4
6,4:sel=4
5,3:sod=4
6,3:sue=4
6,3:use=4
5,5:de=3
5,5:do=3
5,4:ed=3
6,5:el=3
5,4:od=3
6,4:oes=3
6,3:ose=3
6,3:see=3
6,4:see=3
6,5:es=2
6,4:oe=2
